test/runtests.jl

```diff
@@ -253,6 +253,14 @@ end
 @test sprint(JSON.print, Float64) == string("\"Float64\"")
 @test_throws ArgumentError sprint(JSON.print, JSON)
 
+# test for issue #90 - Date/DateTime
+if VERSION >= v"0.4.6-pre+18"
```

Comment:
Should've asked this before, now CI is going to fail...

What's the convention around versioning within the code? `Dates` came in at 0.4 (?), but I'm developing on nightly `0.4.6`, so do I put the version like I did right here (`"0.4.6-pre+18"`) or do put `0.4` (since presumably, this code would've worked as soon as the `Dates` type was merged into base)? This PR fails for 0.3 because there is no `Dates` type given the current state of my submission.